platforms/unix/vm/sqUnixMain.c

```diff
@@ -1819,7 +1817,8 @@ static void parseArguments(int argc, char **argv)
       if (n == 0)			/* option not recognised */
 	{
 	  fprintf(stderr, "unknown option: %s\n", argv[0]);
-	  usage();
+	  usage(1);
+	  /*NOTREACHED*/
```

Comment:
Same as above, refactor exit out of usage() ?   i.e.  here...
     usage(); 
     exit(1)